tests/app/main/views/test_index.py

```diff
@@ -35,6 +24,31 @@ def it_should_show_future_events_in_carousel(self, client, sample_future_events,
                 # expect the other events to be after an intro course if they have an image
                 assert carousel_items[i + 1].text.strip() == other_events[i]['title']
 
+    def it_should_show_future_events_in_cards(self, client, sample_future_events, sample_articles_summary):
+        response = client.get(url_for(
+            'main.index'
+        ))
+        page = BeautifulSoup(response.data.decode('utf-8'), 'html.parser')
+        content = page.find("h4", {"class": "card-title"}).string
+
+        intro_course = [e for e in sample_future_events if e['event_type'] == 'Introductory Course'][0]
+
+        assert content == intro_course['title']
+
+    """ def it_should_show_main_featured_articles
```

Comment:
As the branch is called `stubs-and-tests` are you intending to finish writing up the tests?

Reply:
This branch can be renamed just to stubs as those tests are just pseudocode for now. Will raise another PR for the actual tests tomorrow

Reply:
Could you rename the branch or create a new branch and cherry pick the commits into the new branch and raise a PR.